plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/core/ResetBeanFactoryCaches.java

```diff
@@ -31,16 +31,22 @@
 import java.util.List;
 import java.util.Set;
 
+/**
+ * this class will be removed, now it is not used.
+ */
 public class ResetBeanFactoryCaches {
     private static final AgentLogger LOGGER = AgentLogger.getLogger(ResetBeanFactoryCaches.class);
 
+
     public static void reset(DefaultListableBeanFactory beanFactory) {
-        resetEmbeddedValueResolvers(beanFactory);
-        resetBeanPostProcessors(beanFactory);
-        resetBeanFactoryPostProcessors(beanFactory);
+        // fixme 为什么要清空EmbeddedValueResolvers
+//        resetEmbeddedValueResolvers(beanFactory);
+        // fixme 为什么要清空BeanPostProcessors
+//        resetBeanPostProcessors(beanFactory);
+//        resetBeanFactoryPostProcessors(beanFactory);
     }
 
-    private static void resetEmbeddedValueResolvers(DefaultListableBeanFactory beanFactory) {
+    public static void resetEmbeddedValueResolvers(DefaultListableBeanFactory beanFactory) {
```

Comment:
It looks like useless and is replaced by org.hotswap.agent.plugin.spring.core.BeanFactoryProcessor#resetEmbeddedValueResolvers